vignettes/wrapping-apis.Rmd

```diff
@@ -744,11 +782,11 @@ You can try it out by running this code:
 token <- oauth_flow_auth_code(client, auth_url = "https://github.com/login/oauth/authorize")
 ```
 
-This flow can't be used inside a vignette because it's designed specifically for interactive use: it will open a webpage on Github that requires you to interactively confirm it's ok for this app to use your GitHub account.
+This flow can't be used inside a vignette because it's designed specifically for interactive use: it will open a webpage on GitHub that requires you to interactively confirm it's OK for this app to use your GitHub account.
 
-Other flows provide different ways getting the token:
+Other flows provide different ways of getting the token:
 
--   `req_oauth_client_credentials()` is used to allow the client to perform actions on its own behalf (instead of on behalf of some other user).
+-   `req_oauth_client_credentials()` is used to allow the client to perform actions on its own behalf (instead of on behalf of some other user). JENNY COMMENT HERE
```

Comment:
I perceive the term "service account (token)" to be common enough to mention here. That's what I think this is about, yeah?

Reply:
You mean explicitly connect this function to "service" accounts" or explain more what a service account is?

Reply:
Yeah, I just mean

> explicitly connect this function to "service" accounts"